packages/@o3r/test-helpers/src/prepare-test-env.ts

```diff
@@ -143,11 +152,15 @@ export async function prepareTestEnv(folderName: string, options?: PrepareTestEn
 
   return {
     workspacePath,
-    projectPath,
-    projectName,
+    applicationPath,
+    appName,
+    libraryPath,
+    libName,
     isInWorkspace,
-    untouchedProject,
-    untouchedProjectPath,
+    untouchedApp,
+    untouchedAppPath,
+    untouchedLib,
+    untouchedLibPath,
```

Comment:
as far as I can tell we'll never have to make different check between `untouchedApp` and `untouchedLib`. We always want to check that they are untouched no matter the test
So I would suggest to simplify the tests to only export an array of paths to be checked as untouched
`[untouchedAppPath, untouchedLibPath]`

we could even consider writing a custom matcher to make it more readable
`expect(gitDiff).not.toHaveTouched(path)` or something